admin-ui/app/locales/pt/translation.json

```diff
@@ -508,6 +508,7 @@
     "scope_type": "Tipo de Escopo",
     "scopes:": "Scopes",
     "scripts": "Scripts",
+    "ldap_servers": "Servidores LDAP",
     "script_type": "Tipo de Script",
```

Comment:
Placing a translation key in an incorrect namespace is bad because it violates the principle of consistency by deviating from the established JSON structure used in other locales, which causes the translation lookup to fail and prevents the UI label from being displayed correctly.